pkg/pkg.mk

```diff
@@ -21,8 +21,8 @@ $(PKG_BUILDDIR)/.git-downloaded:
 
 clean::
 	@test -d $(PKG_BUILDDIR) && { \
-		git -C $(PKG_BUILDDIR) clean -f ; \
-		git -C $(PKG_BUILDDIR) checkout "$(PKG_VERSION)"; \
+		cd $(PKG_BUILDDIR) && git clean -f ; \
+		cd $(PKG_BUILDDIR) && git checkout "$(PKG_VERSION)"; \
```

Comment:
You already changed the directory in line 24. If *PKG_BUILDDIR* is absolute this works, although obsolete. If *PKG_BUILDDIR* is relative it will fail and the checkout wouldn't be invoked.